travel.py

```diff
@@ -36,7 +38,9 @@
 if args.cluster:
     CONNSTR = "couchbase://" + args.cluster
 else:
-    CONNSTR = "couchbase://localhost"
+    # 'db' is an alias that resolves to the couchbase-server docker hostname.
+    # See `docker-compose.yml (line 25)`.
+    CONNSTR = "couchbase://db"
```

Comment:
Nitpick: here and below for username/password, the if block seems unnecessary as you could set default in ArgParser instead. (But I realise this was in original, and maybe explicit is more readable...)

Reply:
I think I will leave this as explicit, prefer it to be more readable